docs/pages/development/getting-started.md

```diff
@@ -17,11 +17,11 @@ If you have used Expo before, especially with the Managed workflow, [config plug
 
 <Terminal cmd={[
   "# Only if you don't already have a Managed Workflow project",
-  "$ expo init",
+  "$ npx create-expo-app",
   "",
   "# Install development client",
   "$ expo install expo-dev-client"
-]} cmdCopy="expo init && expo install expo-dev-client" />
+]} cmdCopy="npx create-expo-app && expo install expo-dev-client" />
```

Comment:
there is no way that this command works because it doesn't cd into the directory before running `expo install` 👀

Reply:
Ya I guess the old one didn't either then.